configure.ac

```diff
@@ -64,6 +64,22 @@ AC_ARG_WITH([state-file-path],
 AC_DEFINE_UNQUOTED([STATEFILE], "$STATE_FILE_PATH")
 AC_SUBST(STATE_FILE_PATH)
 
+AC_ARG_WITH([default-mail-command-path],
+  AC_HELP_STRING([--with-default-mail-command-path=PATH],
+                 [path to default mail command (based on config.h if not given)]),
```

Comment:
it is not "path", but "command", i.e the default value is "path" + "arguments": `/bin/mail -s`, so `--with-default-mail-command` would be more appropriate.